app/models/Talk.php

```diff
@@ -1,9 +1,17 @@
 <?php
 
+use Illuminate\Database\Eloquent\SoftDeletes;
+
 class Talk extends UuidBase
 {
+    use SoftDeletes;
+
     protected $table = 'talks';
 
+    protected $dates = [
+      'delted_at'
```

Comment:
Typo here (but that won't matter when we switch to archive instead of soft delete)